### PR TITLE
[HRIS - 433] - [BUGTASK] Notification Filter List lacking "Change Shift"

### PR DESCRIPTION
**Pull request by jessrey-asterisk**
## Issue Link

[Backlog Link](https://framgiaph.backlog.com/view/HRIS-433)

## Definition of Done

- [x] Users can select Change Shift from the notification filter dropdown

## Notes

for here, only mind the latest commit. I didn't think about going back to the main branch then branching it from there. I didn't know that the commits from the past branch would be attached also. 

the error in pre-generating in the front-end is for prettier

## Pre-condition

Run 'npm run dev' on client
Run 'dotnet run' on api

## Expected Output

It should show the "Change Shift" Option in the Dropdown for the Notification Filter

## Screenshots/Recordings

Notification Dropdown
image

### CODE REVIEW

**Review by Fontilllas — COMMENTED** [review]
Please resolve the Prebuild check and build / Build frontend

**Review by Fontilllas — APPROVED** [review]

**Review by JohnlynDC — APPROVED** [review]
Approved

- Able to select and display the "Change Shift" on the notification filter dropdown.
Screenshot 2024-11-06 130717

Comments: 0
Testing time: 15 mins
Iterations: 0

File: client/src/pages/notifications.tsx
Comment (Miguel21Monacillo) [inline client/src/pages/notifications.tsx:1]: Please fix the conflict on this file

**Review by Miguel21Monacillo — APPROVED** [review]
-[x] Able to select and display the "Change Shift" on the notification filter dropdown.
image

*I approved this pr, please fix the conflict first for it to be ready for merging*

**Pull request merged**